app/src/test/java/protect/card_locker/ImportExportTest.java

```diff
@@ -418,25 +418,25 @@ public void multipleCardsExportImportWithGroups() throws IOException {
         List<Group> emptyGroup = new ArrayList<>();
 
         List<Group> groupsForOne = new ArrayList<>();
-        groupsForOne.add(DBHelper.getGroup(mDatabase, "group, \"   1"));
+        groupsForOne.add(DBHelper.getGroupByName(mDatabase, "group, \"   1"));
 
         List<Group> groupsForTwo = new ArrayList<>();
-        groupsForTwo.add(DBHelper.getGroup(mDatabase, "group, \"   1"));
-        groupsForTwo.add(DBHelper.getGroup(mDatabase, "group, \"   2"));
+        groupsForTwo.add(DBHelper.getGroupByName(mDatabase, "group, \"   1"));
+        groupsForTwo.add(DBHelper.getGroupByName(mDatabase, "group, \"   2"));
 
         List<Group> groupsForThree = new ArrayList<>();
-        groupsForThree.add(DBHelper.getGroup(mDatabase, "group, \"   1"));
-        groupsForThree.add(DBHelper.getGroup(mDatabase, "group, \"   2"));
-        groupsForThree.add(DBHelper.getGroup(mDatabase, "group, \"   3"));
+        groupsForThree.add(DBHelper.getGroupByName(mDatabase, "group, \"   1"));
+        groupsForThree.add(DBHelper.getGroupByName(mDatabase, "group, \"   2"));
+        groupsForThree.add(DBHelper.getGroupByName(mDatabase, "group, \"   3"));
 
         List<Group> groupsForFour = new ArrayList<>();
-        groupsForFour.add(DBHelper.getGroup(mDatabase, "group, \"   1"));
-        groupsForFour.add(DBHelper.getGroup(mDatabase, "group, \"   2"));
-        groupsForFour.add(DBHelper.getGroup(mDatabase, "group, \"   3"));
+        groupsForFour.add(DBHelper.getGroupByName(mDatabase, "group, \"   1"));
+        groupsForFour.add(DBHelper.getGroupByName(mDatabase, "group, \"   2"));
+        groupsForFour.add(DBHelper.getGroupByName(mDatabase, "group, \"   3"));
 
         List<Group> groupsForFive = new ArrayList<>();
-        groupsForFive.add(DBHelper.getGroup(mDatabase,"group, \"   1"));
-        groupsForFive.add(DBHelper.getGroup(mDatabase, "group, \"   3"));
+        groupsForFive.add(DBHelper.getGroupByName(mDatabase,"group, \"   1"));
+        groupsForFive.add(DBHelper.getGroupByName(mDatabase, "group, \"   3"));
 
         DBHelper.setLoyaltyCardGroups(mDatabase, 1, groupsForOne);
         DBHelper.setLoyaltyCardGroups(mDatabase, 2, groupsForTwo);
```

Comment:
Can't reply to the real line (461), but the importer reports failing for some reason